src/app/services/user.js

```diff
@@ -715,6 +723,33 @@ module.exports = (Model, App) => {
   };
 
   const sendEmailVerification = async (user) => {
+    const [mailLimit] = await Model.mailLimit.findOrCreate({
```

Comment:
where is the limit being reset?

Reply:
Added the reset

Reply:
That reset is not going to work. If the limit is reached, no matter the day, you will always throw an Error before checking the day. Please check that before checking the count vs limit and then reject or adjust the limit and allow.